styles.css

```diff
@@ -0,0 +1,712 @@
+.majisafiP {
+  color: #8f8f91;
+  display: flex;
+  font-size: 14px;
+  text-align: center;
+  padding: 20px;
+  justify-content: center;
+}
+
+.majisafi {
+  width: 20%;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+}
+
+.group-photo {
+  padding: 24px;
+}
+
+.grp {
+  width: 94%;
+  height: 69%;
+  margin-left: auto;
+  padding: 5px 0;
+  margin-right: auto;
+  display: flex;
+}
+
+.pastphotos {
+  background-color: #fff;
+  padding: 50px 10px;
+}
+
+.pastp {
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+}
+
+.pag {
+  font-size: 12px;
+  color: #757370;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  padding-top: 12px;
+}
+
+.image_wrapper {
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  background: rgba(237, 81, 81, 0.5);
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  justify-content: center;
+  margin: 37px 54px;
+  flex-direction: column;
+}
+
+.overlay_2 {
+  inset: 0;
+}
+
+.img-title {
+  color: #fff;
+  font-size: 30px;
+  display: flex;
+  justify-content: center;
+}
+
+.p {
+  color: #fff;
+  font-size: 15px;
+}
+
+.upper-nav {
+  display: none;
+}
+
+.heading31 {
+  display: none;
+}
+
+.hrprog {
+  display: none;
+}
+
+.flex .logo {
+  display: none;
+  gap: 10%;
+}
+
+.sub {
+  color: #979696;
+  font-size: 16px;
+  font-weight: bold;
+  margin-top: 10px;
+}
+
+@media screen and (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    margin-left: 5%;
+    width: 100%;
+  }
+
+  .nav-menu {
+    position: inherit;
+    flex-direction: row;
+    backdrop-filter: blur(0);
+    padding-top: 20px;
+    padding-right: 0;
+    background-color: transparent;
+    text-align: center;
+    transition: 0.3s;
+    width: 70%;
+    gap: 35px;
+    color: #344563;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 27px;
+    line-height: 44px;
+    display: flex;
+  }
+
+  .nav-link {
+    display: flex;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    transition: 0.7s ease;
+  }
+
+  #nav1 {
+    color: #ec5242;
+  }
+
+  .nav-item {
+    padding-left: 0;
+    margin-right: 0;
+  }
+
+  .upper-nav {
+    display: block;
+    background-color: #383839;
+  }
+
+  .top {
+    display: flex;
+    gap: 17px;
+    padding: 5px;
+    margin-left: 56%;
+  }
+
+  .pa {
+    color: white;
+    font-size: 12px;
+  }
+
+  #fb {
+    width: 16px;
+    height: 16px;
+  }
+
+  #twitter {
+    width: 16px;
+    height: 16px;
+  }
+
+  .GE {
+    padding-left: 0;
+    height: 25px;
+    color: #ec5242;
+    border: 3px solid #ec5242;
+  }
+
+  #ge {
+    color: #ec5242;
+    font-weight: 900;
+    padding: 0 15px;
+  }
+
+  #logo {
+    width: 40px;
+    margin-left: 120px;
+    margin-top: 11px;
+    height: 40px;
+  }
+
+  .flex {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
- [  ] I kindly suggest that you ensure that the `nav bar` has a white background☺️. This is to ensure that you try as much as possible to stick to the original design and have improved UI 👍.

Your design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/c7796e54-e7ff-4365-95b4-9a2ea30ffe87)

Original design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/241668a9-9a7f-414d-93a9-83f693686192)